xc/xc7/tests/soc/litex/generate.py

```diff
@@ -20,7 +20,7 @@ class BaseSoC(SoCCore):
     def __init__(
             self,
             toolchain="vivado",
-            sys_clk_freq=int(60e6),
+            sys_clk_freq=int(100e6),
```

Comment:
It probably isn't a good idea to increase the sys clock freq.  I don't believe that VPR can achieve 100 MHz.  @acomodi ?

Reply:
With latest LiteX we have a minimum of 80MHz, below which DDR does not work even in a full Vivado flow. I think that we can set it to 80MHz.

80MHz worked on HW with symbiflow, but might not work on every single arty.